app/templates/server/config/seed(models).js

```diff
@@ -17,7 +17,7 @@ var User = sqldb.User;<% } %><% } %>
   })<% } %>
   .then(() => {
     <% if (filters.mongooseModels) { %>Thing.create({<% }
-       if (filters.sequelizeModels) { %>Thing.bulkCreate([{<% } %>
+       if (filters.sequelizeModels) { %>return Thing.bulkCreate([{<% } %>
```

Comment:
Are these changes completely necessary to make it work? We are going to want to return *all* promises in this file. I'm probably going to refactor this file.